server/tests/run_parallel.py

```diff
@@ -50,27 +56,27 @@ def sendevent(self, name, **kwargs):
     @pytest.hookimpl
     def pytest_internalerror(self, excrepr):
         formatted_error = str(excrepr)
-        self.sendevent("internal_error", formatted_error=formatted_error)
+        self.sendevent(EVT_INTERNAL_ERROR, formatted_error=formatted_error)
 
     @pytest.hookimpl
     def pytest_collection_finish(self, session: pytest.Session):
-        self.sendevent("collection_finish", total_tests_count=len(session.items))
+        self.sendevent(EVT_COLLECTION_FINISH, total_tests_count=len(session.items))
 
     @pytest.hookimpl
     def pytest_sessionstart(self, session):
-        self.sendevent("workerready")
+        self.sendevent(EVT_WORKER_READY)
 
     @pytest.hookimpl
     def pytest_sessionfinish(self, exitstatus):
-        self.sendevent("workerfinished", exitstatus=exitstatus)
+        self.sendevent(EVT_WORKER_FINISHED, exitstatus=exitstatus)
 
     @pytest.hookimpl
     def pytest_runtest_logreport(self, report: pytest.TestReport):
         # Ignore tests skipped for being out of the job's tests slice
         if ("test_out_of_slice", True) in report.user_properties:
             return
         self.sendevent(
-            "testreport",
+            EVT_TEST_REPORT,
```

Comment:
This value is being changed (here and also below). Can't this have an impact elsewhere?

Reply:
https://github.com/Scille/parsec-cloud/pull/5131#discussion_r1317278886